pulsar-io/file/src/main/java/org/apache/pulsar/io/file/utils/ZipFiles.java

```diff
@@ -44,19 +43,16 @@
     /**
      * Returns true if the given file is a gzip file.
      */
-   @SuppressWarnings("deprecation")
-   public static boolean isZip(File f) {
-
-       InputStream input = null;
+    public static boolean isZip(File f) {
+        DataInputStream in = null;
         try {
-            DataInputStream in = new DataInputStream(new BufferedInputStream(new FileInputStream(f)));
+            in = new DataInputStream(new BufferedInputStream(new FileInputStream(f)));
```

Comment:
This could be further improved with : 

```java
try (DataInputStream in = new DataInputStream(new BufferedInputStream(new FileInputStream(f))) {
     /// ... 
}
```

Reply:
Thank you @merlimat , I pushed 2d89507f13e79d265d3c2fd407f26ff3e5577195